tools/PTE/pte-execRequest.js

```diff
@@ -53,6 +51,7 @@ var evtType = 'FILTEREDBLOCK';        // event type: FILTEREDBLOCK|CHANNEL, defa
 var evtTimeout = 120000;              // event timeout, default: 120000 ms
 var evtListener = 'BLOCK';            // event listener: BLOCK|TRANSACTION, default: BLOCK
 var evtLastRcvdTime = 0;              // last event received time
+var evtValidCode = 0;                 // event valid code (channel block only)
```

Comment:
This code is defined in fabric repo somewhere, right? what and where is that? You are looking inside a message; I was suggesting that this be defined here as a constant, yes, but also the comment should mention the API (or "source of truth"), since we are not including the API library of definitions directly in the build. The point is we are not making up the value zero ourselves, but are depending on the fabric code to set it and we are checking it here.

Reply:
add source of the valid code in the comment

Reply:
Re: Also, we should probably include this counter for "invalid" received events in the pteReport output.

The invalid event count is included in the pteReport output